### PR TITLE
Use optional pretty URL

### PR DESCRIPTION
**Pull request by TomTurnschuh**
Based on and closes #605.

A new app allowing to enable "pretty URLs" (name: `nc-prettyURL`).

### CODE REVIEW

**Comment by TomTurnschuh** [comment]
Could somebody please test and comment on it? I didn't manage to do so.
Especially the task to get the default value of ´DOMAIN_´ might be tricky.
@nachoparker @enoch85

File: etc/ncp-config.d/nc-prettyURL [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+You are not sure about it? Check the domain in your browser."
+
+NCDIR=/var/www/nextcloud
+OCC="$NCDIR/occ"
+
+install() { :; }
+
+configure() 
+{
+  chown -R www-data:www-data $NCDIR
```
[the remark this answers is not in the document]
Comment (TomTurnschuh) [reply etc/ncp-config.d/nc-prettyURL]: Thanks, I wasn't sure about it.

File: etc/ncp-config.d/nc-prettyURL [hunk 2]
```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#
+# Activate/deactivate SSH
```
[the remark this answers is not in the document]
Comment (TomTurnschuh) [reply etc/ncp-config.d/nc-prettyURL]: Is updated

**Comment by nachoparker** [comment]
Looks good to me. I will test it and merge it soon.

Thanks!

**Comment by nachoparker** [comment]
Hi, please see my comments in this commit https://github.com/nextcloud/nextcloudpi/commit/a7e81596ec6284e7d3535808e1f3210d35a16cd2

**Comment by nachoparker** [comment]
Tested. It works after my fixes in https://github.com/nextcloud/nextcloudpi/commit/a7e81596ec6284e7d3535808e1f3210d35a16cd2

**Pull request closed without merging**